core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -57,6 +57,7 @@
     public static final String OPTIMIZE_HASH_GENERATION = "optimize_hash_generation";
     public static final String JOIN_DISTRIBUTION_TYPE = "join_distribution_type";
     public static final String JOIN_MAX_BROADCAST_TABLE_SIZE = "join_max_broadcast_table_size";
+    public static final String JOIN_MULTI_CLAUSE_INDEPENDENCE_FACTOR = "join_multi_clause_selectivity_weight";
```

Comment:
I would just name it `join_conjunction_independence_factor` for consistency. All this properties should land in separate system session provider anyway.

btw: you have `"join_multi_clause_selectivity_weight"` (forgot to rename)

Reply:
renamed to `join_multi_clause_independence_factor`